hermes/templates/python/template.py.tpl

```diff
@@ -235,8 +235,8 @@ class DefaultSyntaxErrorHandler:
         return error
     def unexpected_eof(self):
         return self._error("Error: unexpected end of file")
-    def excess_tokens(self):
-        return self._error("Finished parsing without consuming all tokens.")
+    def excess_tokens(self, next_token):
+        return self._error("Finished parsing without consuming all tokens.  Next token was {next}".format(next=next_token))
```

Comment:
If you don't want to change all of the templates, you can revert the line above and then supply your own implementation of this class within PyWDL which will print the message the way you want it.  That's the route I'm going to take next time I do work on PyWDL (I'll check with you first though so I don't step on your feet)